ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-rollback-stable.yaml

```diff
@@ -101,6 +101,17 @@ tests:
     - ref: cucushift-chainupgrade-toimage
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-ovn-hypershift-inplace-rollback-f7
+  cron: 2 15 1,8,15,22 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRA_ARGS: --node-upgrade-type=InPlace
+      TEST_FILTERS: ~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&
```

Comment:
For hypershift cluster, is it only run against a stable payload, right?

Reply:
okay, move to *4.16__amd64-rollback-nightly.yaml